### DISCUSSION TITLE
setBaseColorFactor doesn’t work

### DISCUSSION THREAD

**Discussion by pragavame** [google/model-viewer#3884] [Q&A]
Hey!

I can't change color with my color picker.
But when I took the standard example, I realized that it also does not change the RGB color.

`<button data-color="0,1,0,1">Green</button>`
->
`<button data-color="50,184,72,1">Green</button>`

And it didn't work

Example: https://modelviewer.dev/examples/scenegraph/#changeColor

Then I tried to change the function itself and it didn’t work either and the color doesn’t change

`material.pbrMetallicRoughness.setBaseColorFactor([50, 184, 72]);`

and

`material.pbrMetallicRoughness.setBaseColorFactor([50, 184, 72, 1]);`

**Comment by elalish** [comment]
That example is working for me. Can you please post your repro link like the issue template asks? Did you look at your JS console?
